src/vs/workbench/contrib/chat/browser/agentSessions/localAgentSessionsProvider.ts

```diff
@@ -158,7 +158,13 @@ export class LocalAgentsSessionsProvider extends Disposable implements IChatSess
 	private async getHistoryItems(): Promise<ChatSessionItemWithProvider[]> {
 		try {
 			const allHistory = await this.chatService.getHistorySessionItems();
-			return coalesce(allHistory.map(history => this.toChatSessionItem(history)));
+			return coalesce(allHistory.map(history => {
+				const sessionItem = this.toChatSessionItem(history);
+				return sessionItem ? {
+					...sessionItem,
+					history: true
+				} : undefined;
+			}));
 		} catch (error) {
```

Comment:
can you please add a `todo@bpasero comment` to remind me to clean this up later.